### PR TITLE
Add tiling example

### PR DESCRIPTION
**Pull request by mcoughlin**
Quick example to make a tiling on the sphere

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ To install with [Pip]:
 5.  To generate an animated visualization for this observing plan, run the
     following command:
 
-        $ dorado-scheduling-animate exaples/6.fits examples/6.ecsv examples/6.gif
+        $ dorado-scheduling-animate examples/6.fits examples/6.ecsv examples/6.gif
```
Comment (lpsinger) [inline README.md:140]: Good catch! I just fixed that on master, so please rebase.

File: dorado/scheduling/scripts/tile.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                   help='in degrees.')
+    p.add_argument('scale', default=0.8,
+                   type=float,
+                   help='scale for overlap.')
+    p.add_argument('output', metavar='tess.dat', type=FileType('wb'),
+                   help='Output filename')
+    return p
+
+def tesselation_spiral(FOV_size, scale=0.80):
+    FOV = FOV_size*FOV_size*scale
```
Comment (lpsinger) [inline dorado/scheduling/scripts/tile.py:32]: Please fix these lint errors.

File: dorado/scheduling/scripts/tile.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+    area_of_sphere = 4*np.pi*(180/np.pi)**2
+    n = int(np.ceil(area_of_sphere/FOV))
+    print("Using %d points to tile the sphere..."%n)
```
Comment (lpsinger) [inline dorado/scheduling/scripts/tile.py]: Suggest removing print statement.

File: dorado/scheduling/scripts/tile.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    z = np.linspace(1 - 1.0 / n, 1.0 / n - 1, n)
+    radius = np.sqrt(1 - z * z)
+
+    points = np.zeros((n, 3))
+    points[:,0] = radius * np.cos(theta)
+    points[:,1] = radius * np.sin(theta)
+    points[:,2] = z
+
+    ra, dec = hp.pixelfunc.vec2ang(points, lonlat=True)
+    return ra, dec
```
Comment (lpsinger) [inline dorado/scheduling/scripts/tile.py]:
```suggestion
    coords = SkyCoord(radius, theta * u.rad, z, representation_type='cylindrical')
    coords.representation_type = 'unitspherical'
    return coords
```

File: dorado/scheduling/scripts/tile.py [hunk 5]
```diff
@@ -0,0 +1,62 @@
+#
+# Copyright © 2020 United States Government as represented by the Administrator
+# of the National Aeronautics and Space Administration. No copyright is claimed
+# in the United States under Title 17, U.S. Code. All Other Rights Reserved.
+#
+# SPDX-License-Identifier: NASA-1.3
+#
+"""Create a tesselation."""
+import logging
```
Comment (lpsinger) [inline dorado/scheduling/scripts/tile.py:9]:
```suggestion
```

Remove unused import.

File: dorado/scheduling/scripts/tile.py [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+import healpy as hp
```
Comment (lpsinger) [inline dorado/scheduling/scripts/tile.py:10]:
```suggestion
from astropy.coordinates import SkyCoord
```

**Review by lpsinger — CHANGES_REQUESTED** [review]
Nice, but we'll have to move this to a module eventually.

**Comment by lpsinger** [comment]
Would you mind plotting the output of this please, to check?

**Comment by lpsinger** [comment]
Here's what this looks like near the north celestial pole. It has a lot of overlap. Does this look correct?

image_2021_03_12T01_05_54_932Z

**Comment by mcoughlin** [comment]
Yeah I just added one that I think behaves a bit better at the poles. Can we add that script to tiles.py for the plotting?

**Comment by lpsinger** [comment]
What is the provenance of these algorithms? Have they been used for previous surveys?

**Comment by mcoughlin** [comment]
I have used a variation on these to produce tilings for various telescopes, including DECam and others. I usually tune the overlaps by hand a bit for the various surveys to get some reasonable amount of overlap, which also means needing to play around a bit close to the poles.

**Comment by lpsinger** [comment]
Would you please add docstrings with references to the journal articles where these grids are introduced, described, or used?

File: dorado/scheduling/scripts/tile.py [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+    z = np.linspace(1 - 1.0 / n, 1.0 / n - 1, n)
+    radius = np.sqrt(1 - z * z)
+
+    coords = SkyCoord(radius, theta * u.rad, z,
+                      representation_type='cylindrical')
+    coords.representation_type = 'unitspherical'
+    return coords
+
+
+def tesselation_packing(FOV_size, scale=0.97):
```
Comment (lpsinger) [inline dorado/scheduling/scripts/tile.py:49]: This is a pretty generic name.

**Comment by lpsinger** [comment]
I'm ready to merge this as soon as legal processes your CLA. So sorry about the red tape.

**Comment by lpsinger** [comment]
I added geodesic grids: https://dorado-scheduling.readthedocs.io/en/latest/reference/tesselation.html

**Comment by mcoughlin** [comment]
awesome leo!

**Comment by lpsinger** [comment]
They finally got your CLA processed. I'd like to see this unified with the new `dorado.scheduling.tesselation` module, but since I added that after you had opened this PR, I could merge this as is and do the refactoring myself. What would you prefer?

**Comment by mcoughlin** [comment]
Let's get this merged and then I can help as necessary this weekend when I have more time. Thanks Leo.

**Pull request merged**